mkdocs.yml

```diff
@@ -121,6 +122,7 @@ nav:
   - sketch-specification.md
   - library-specification.md
   - platform-specification.md
+  - Pluggable Discovery specification: pluggable-discovery-specification.md
```

Comment:
Was there a specific reason for the placement here? I'm thinking that, since more platform developers will be interested in writing a package index than will be interested in writing a pluggable discovery tool, the order of the two should be swapped. But there might be another logical reason for the current order that I'm missing.

Reply:
I just thought that since it's more related to the platforms' specification it would make sense for it to come after those. I can move below the package index specification if you think it's better.

Reply:
After a bit more consideration, I think it's OK where it is.